README.md

```diff
@@ -61,6 +61,46 @@ If you get the error:
 
 Be sure to upgrade to `@symfony/webpack-encore` version 1.0.0 or higher.
 
+## The controllers.json File
+
+The bridge works by reading a `controllers.json` file that is automatically
+updated by Symfony Flex whenever you download a UX-powered package. For
+example, after running `composer require symfony/ux-dropzone`, it will
+look like this:
+
+```json
+{
+    "controllers": {
+        "@symfony/ux-dropzone": {
+            "dropzone": {
+                "enabled": true,
+                "webpackMode": "eager",
+                "autoimport": {
+                    "@symfony/ux-dropzone/src/style.css": true
+                }
+            }
+        }
+    },
+    "entrypoints": []
+}
+```
+
+Each item under `controllers` will cause a Stimulus controller to be
+registered with a specific name - in this case the controller would
+be called `symfony--ux-dropzone--dropzone` (the `/` becomes `--`).
+
+By default, the new controller will always be included in your
+JavaScript package. You can control that with the `webpackMode` option,
+ordered from least to most lazy:
+
+* `webpackMode: 'eager'`: controller & dependencies are included in the JavaScript
+  that's downloaded when the page is loaded.
+* `webpackMode: 'lazy'`: controller & dependencies are isolated into a separate
```

Comment:
This is (fortunately) not invented by us: we're leveraging native webpack functionality: https://webpack.js.org/api/module-methods/#magic-comments

The one exception is the new `lazy-controller`: that's an "add on" by us.

Reply:
But `webpackMode: 'lazy'` is meant to say "load the file when necessary". The issue is that the way our generated file looks like, all controllers which are not registered with your new feature are necessary immediately. This means that setting `webpackMode: 'lazy'` in the Symfony UX config will **not** result in the same behavior that what you would intend from `webpackMode: 'lazy'` when using dynamic imports in webpack directly.

Reply:
also, if we start having our own values, I would strongly suggest using a different name than `webpackMode` for the config key in Symfony UX.

Reply:
I hear your point, however I don't agree that there is a real issue here.

However, we could change to `load: 'lazy'` or `load: 'eager'` as the only two values (and deprecated the old ones) - the `load: 'lazy'` would be equivalent to the `lazy-controller`, as it's probably more useful than just loading everything immediately, but via async (which is what `webpackMode: 'lazy'` does currently). This will just cause a bit more of a headache, as we'll need to update all the UX packages for the new key and probably keep `webpackMode` as a deprecated option (even though this us still experimental, I'd like to minimize annoying breaks).

Reply:
well, the thing is, the existing `lazy` value in symfony/ux is useless (IMO) and does not do what users would expect (as it does not lazy-load on usage). So I agree about keeping only 2 values.

Reply:
I agree, we should probably just replace the "lazy" behavior with this new one completely.

Reply:
Yes - *and* then stof is suggesting we stop using webpackMode as it’s not really that anymore and just use something like: “load”.